packages/framework-provider-azure-infrastructure/package.json

```diff
@@ -32,7 +32,8 @@
     "copyfiles": "^2.3.0",
     "ms-rest-azure": "^3.0.0",
     "mustache": "^4.0.1",
-    "needle": "^2.5.0"
+    "needle": "^2.5.0",
+    "uuid": "^8.3.2"
```

Comment:
What's the difference between this `uuid` and `@types/uuid` from devDependencies below?

`"uuid": "^8.3.2"` vs `"@types/uuid": "^8.3.0",`

Also mustache:
`"mustache": "^4.0.1"` vs `"@types/mustache": "^4.1.0"`

Reply:
"uuid" is the runtime dependency. It is the package itself. However, it is a package done in Javascript, not in Typescript, so when compiling we get errors because Typescript don't know anything about the types of the things defined in "uuid". The solution for that is to provide an extra package with all the types definitions (just the definitions, they go in a separate .d.ts file). And those packages are normally under the @types scope.

So when using a package in a Typescript project that was developed using Javascript, you need to also include the type definition package, which in this case is `@types/uuid` (the same with mustache)

**Regarding your question about `-lock.json` files**

 I think you are correct. I will upload them after fixing the tests ;-)